### PR TITLE
add exclude options to Metabase crawler [sc-27133]

### PR DESCRIPTION
**Pull request by alyiwang**
### 🤔 Why?

To be able to filter Metabase assets based on collection path

### 🤓 What?

- add asset_filter in metabase config
- support filtering assets based on collection path (directory)

### 🧪 Tested?

tested against Metaphor instance.

### ☑️ Checks

- [x] My PR contains actual code changes, and I have updated the version number in `pyproject.toml`.

### CODE REVIEW

File: metaphor/metabase/README.md [hunk 1]
```diff
@@ -39,6 +39,27 @@ database_defaults:
...
 See [Output Config](../common/docs/output.md) for more information.
 
+#### Excluding Directories
+
+You can specify the directories (collection paths) to be included / excluded by the connector. By default all assets are included.
+
+To specify the directories to include / exclude, use the following field:
+
+```yaml
+directories_filter:
```
Comment (mars-lan) [inline metaphor/metabase/README.md:49]: Consider renaming this to either `directory_filter` or just `filter` for consistency with https://github.com/MetaphorData/connectors/blob/main/metaphor/common/docs/filter.md

Comment (alyiwang) [reply metaphor/metabase/README.md:49]: updated

Comment (mars-lan) [reply metaphor/metabase/README.md:49]: Pls also update README.

**Review by usefulalgorithm — APPROVED** [review]
LGTM!

**Comment by mars-lan** [comment]
Please create a follow-up PR to fix https://github.com/MetaphorData/connectors/pull/887#discussion_r1661711686

**Pull request merged**